frontend/src/pages/meditation/meditation-sidebar/meditation-sidebar.tsx

```diff
@@ -1,33 +1,53 @@
+import { type SetURLSearchParams } from 'react-router-dom';
```

Comment:
Move to common types

Reply:
done

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/3716ece6-73aa-405c-a197-5cbc9e33e0db)